src/main/java/kea/alog/project/domain/topic/service/TopicServiceImpl.java

```diff
@@ -65,10 +70,28 @@ private Sort getSort(TopicSortType sortType) {
 
     @Override
     public TopicDto findOne(Long projectPk, Long topicPk) {
-        Optional<Topic> topic = topicRepository.findByPkAndProjectPk(topicPk, projectPk);
-        if (!topic.isPresent()) {
-            throw new EntityNotFoundException("ENTITY_NOT_FOUND");
-        }
-        return topicMapper.topicToDto(topic.get());
+        Topic topic = topicRepository.findByPkAndProjectPk(topicPk, projectPk)
+                                     .orElseThrow(() ->
+                                         new EntityNotFoundException("ENTITY_NOT_FOUND"));
+
+        return topicMapper.topicToDto(topic);
+    }
+
+    @Transactional
+    @Override
+    public CreateTopicResponseDto create(Long projectPk,
+        CreateTopicRequestDto createTopicRequestDto) {
+        Project project = projectRepository.findById(projectPk).orElseThrow(
+            () -> new EntityNotFoundException("ENTITY_NOT_FOUND"));
+
+        Topic topic = topicRepository.save(Topic.builder()
+                                                .project(project)
+                                                .name(createTopicRequestDto.getName())
```

Comment:
There is no build(); at the end. is it work?

Reply:
It's in line 93!

Reply:
K, I  found it. It was hidden lol :)